src/test/ui/const-generics/min_const_generics/forbid-non-static-lifetimes.rs

```diff
@@ -0,0 +1,24 @@
+#![feature(min_const_generics)]
```

Comment:
I'd test this with revisions like the other tests.

Reply:
Fixed!